src/ostorlab/runtimes/local/runtime.py

```diff
@@ -170,6 +170,23 @@ def _docker_checks(self):
 
         self._docker_client = docker.from_env()
 
+    def prepare_scan(
+        self, title: str, assets: Optional[List[base_asset.Asset]]
+    ) -> None:
+        """Prepare scan entry in the database.
+
+        Args:
+            title: Scan title.
+            assets: The target asset to scan.
+        """
+
+        console.info("Creating scan entry")
+        if assets is None:
```

Comment:
I would add a unit test for this case.

Reply:
which case man, this is a copy paste (refactoring)